guides/v2.3/graphql/queries.md

```diff
@@ -1,35 +1,164 @@
 ---
 group: graphql
-title: Searches and pagination in GraphQL
+title: Queries
+redirect_from:
+  - /guides/v2.3/graphql/search-pagination.html
 ---
 
-A GraphQL search query can contain the following components:
+A GraphQL query retrieves data from the Magento server in a similar manner as a REST GET call. However, Magento GraphQL queries are not meant to be as comprehensive as their REST counterparts. The main goal of the Magento queries is to allow a mobile app or browser to render a wide variety of information, including the following:
+
```

Comment:
"The main goal of the Magento GraphQL queries..."

I thought maybe being explicit here since we're discussing graphql vs REST.